EIPS/eip-3155.md

```diff
@@ -24,26 +26,12 @@ This EIP has multiple objectives:
 Implementing this EIP in all major clients allows us to create meaningful differential fuzzers that fuzz EVM implementations for the mainnet and all upcoming hardforks.
 It also helps finding differences in execution quickly in the case of a chain split.
 
-Due to the slight differences in output the following fields currently need to be dropped when comparing tracing outputs: $GasCost$, $MemorySize$, $RefundCounter$, $ReturnStack$, and $ReturnData$. This reduces the usefulness of tracing significantly. Creating a common standard would allow to add these fields thus increasing the surface that can be tested by a fuzzer.
-
-This EIP will enable users to create better differential fuzzing infrastructure to compare the EVM implementations of all major Ethereum clients against each other. This could help to find new unforeseen bugs that are currently present in the client implementations. 
+This EIP will enable users to create better differential fuzzing infrastructure to compare the EVM implementations of all major Ethereum clients against each other. 
+This could help to find new unforeseen bugs that are currently present in the client implementations. 
```

Comment:
```suggestion
This could help to find bugs that are currently present in the client implementations. 
```
A bit superfluous with "new unforeseen".. ?